release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,82 @@ vf:fulfills     a owl:ObjectProperty ;
 #        rdfs:range      xsd:boolean ;
 #        rdfs:comment    "The commitment or process is complete or not." .
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceSpecification ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+vf:unit a               owl:ObjectProperty ;
+        rdfs:label      "unit" ;
+        rdfs:domain     vf:ResourceSpecification ;
+        rdfs:range      qudt:unit ;
+        rdfs:comment    "The unit expected for this resource specification as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, skill, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceSpecification ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a          owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
```

Comment:
I don't think this comes as fundamental and possibly we can take advantage of PROV https://www.w3.org/TR/prov-primer/